lib/matplotlib/scale.py

```diff
@@ -458,6 +458,69 @@ def get_transform(self):
         return self._transform
 
 
+class PowerTransform(Transform):
+    input_dims = output_dims = 1
+
+    def __init__(self, gamma, nonpositive='clip'):
+        super().__init__()
+
+        self.gamma = gamma
+        self._clip = _api.check_getitem(
+            {"clip": True, "mask": False}, nonpositive=nonpositive)
+
+    def __str__(self):
+        return "{}(gamma={}, nonpositive={!r})".format(
+            type(self).__name__, self.gamma, "clip" if self._clip else "mask")
+
+    def transform_non_affine(self, a, gamma):
+        with np.errstate(divide="ignore", invalid="ignore"):
+            out = np.power(a, gamma)
+            if self._clip:
+
+                out[a <= 0] = 0
+            return out
+
+    def inverted(self):
+        return InvertedPowerTransform(self.gamma)
```

Comment:
```suggestion
```suggestion
class PowerTransform(Transform):
    input_dims = output_dims = 1

    def __init__(self, gamma, nonpositive='clip'):
        super().__init__()

        self.gamma = gamma
        self._clip = _api.check_getitem(
            {"clip": True, "mask": False}, nonpositive=nonpositive)

    def __str__(self):
        return "{}(gamma={}, nonpositive={!r})".format(
            type(self).__name__, self.gamma, "clip" if self._clip else "mask")

    def transform_non_affine(self, a):
        with np.errstate(divide="ignore", invalid="ignore"):
            out = np.power(a, self.gamma)
            if self._clip:
                out[a <= 0] = 0
            return out

    def inverted(self):
        return InvertedPowerTransform(self.gamma)
```
```